tests/io/test_file.py

```diff
@@ -61,13 +70,17 @@ def test_staypoints_from_to_postgis(self):
 
     def test_locations_from_to_csv(self):
         orig_file = os.path.join('tests', 'data', 'locations.csv')
+        mod_file = os.path.join('tests','data','locations_mod_columns.csv')
         tmp_file = os.path.join('tests', 'data', 'locations_test.csv')
-        locs = ti.read_locations_csv(orig_file, sep=';')
-        locs.as_locations.to_csv(tmp_file, sep=';',
-                                 columns=['user_id', 'elevation', 'center', 'extent'])
+        mod_plcs = ti.read_locations_csv(mod_file, columns={'geom':'center'},sep=';')
+        plcs = ti.read_locations_csv(orig_file, sep=';')
+        assert mod_plcs.equals(plcs)
+        plcs.as_locations.to_csv(tmp_file, sep=';',
+                              columns=['user_id', 'elevation', 'center', 'extent'])
```

Comment:
change the `plcs` (places) to `locs` (locations). We have changed the wording "places" into "locations".